Cargo.toml

```diff
@@ -9,21 +9,16 @@ homepage = "https://github.com/mackwic/colored"
 repository = "https://github.com/mackwic/colored"
 readme = "README.md"
 keywords = ["color", "string", "term", "ansi_term", "term-painter"]
-rust-version = "1.70"
+rust-version = "1.80"
 
 [features]
 # with this feature, no color will ever be written
 no-color = []
 
-[dependencies]
-lazy_static = "1"
 
 [target.'cfg(windows)'.dependencies.windows-sys]
 version = "0.48"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-]
+features = ["Win32_Foundation", "Win32_System_Console"]
```

Comment:
I'm personally pretty nitpicky about unnecessary noise in diffs.

For example, if I used `git blame` to find who created these features, the blame would pick you if this PR was merged.

Reply:
Removed this change in a force push